Gems/Terrain/Code/Source/TerrainRenderer/TerrainMeshManager.h

```diff
@@ -86,7 +104,9 @@ namespace Terrain
 
         void DrawMeshes(const AZ::RPI::FeatureProcessor::RenderPacket& process, const AZ::RPI::ViewPtr mainView);
 
-        static constexpr int32_t GridSize{ 64 }; // number of terrain quads (vertices are m_gridSize + 1)
+        static constexpr int32_t GridSizeExponent = 6; // 2^6 = 64
+        static constexpr int32_t GridSize{ 1 << GridSizeExponent }; // number of terrain quads (vertices are m_gridSize + 1)
```

Comment:
Any particular reason to change it to this instead of just leaving it as 64? It seems unnecessarily less readable if GridSizeExponent isn't used anywhere else.

Reply:
It used to be used in earlier versions of my code, but no longer is, so I'll switch this back. I was really annoyed there's no constexpr way to do log2 that I could find.